styles.css

```diff
@@ -388,7 +388,7 @@ li {
 }
 
 .project-x-para {
```

Comment:
[OPTIONAL] Kindly fix the alignment of your technologies to be the same as the figma template:

Your design: (technologies not centered: HTML, bootstrap, etc..)
![buttons](https://user-images.githubusercontent.com/111214189/229314643-4e58fb08-705b-488e-ab6d-67fa71f9e66d.png)

Figma template: (technologies centered)
![figmacard](https://user-images.githubusercontent.com/111214189/229314661-14954d90-9842-4117-a82b-1f411b8f200c.png)

This will help showcase to users you have great attention to detail. This was missed by the previous reviewer but I **highly** recommend you fix it to make your portfolio look more professional 👍